tests/test_evaluator_runner.py

```
@@ -0,0 +1,249 @@
+
+    def test_rejected(self, sample_config, tmp_path, capsys):
+        """REJECTED verdict returns 1."""
+        log_file = tmp_path / "test.md"
+        log_file.write_text("test")
+
+        result = _report_verdict("REJECTED", log_file, sample_config)
+        assert result == 1
+        captured = capsys.readouterr()
+        assert "REJECTED" in captured.out
+
+    def test_unknown_verdict(self, sample_config, tmp_path, capsys):
```

Ruff (0.14.13)

Unused method argument: `capsys`

(ARG002)